src/zui/components/ZUIOrgLogoAvatar/index.tsx

```diff
@@ -30,10 +29,13 @@ const ZUIOrgLogoAvatar: FC<ZUIOrgLogoAvatarProps> = ({
   urlBase = '/api',
 }) => {
   return (
-    <Image
+    /* eslint-disable @next/next/no-img-element*/
+    //TODO: Investigate how we can use the Image component from Next.js here
+    <img
       alt="icon"
       height={avatarSizes[size]}
       src={`${urlBase}/orgs/${orgId}/avatar`}
+      style={{ flexShrink: 0 }}
```

Comment:
How does this related to #2744?

Reply:
I think this was added after a conversation with rebeca about something, i cannot say for sure, but i don't think it's directly linked to the problem of the avatar logos so i can def remove it!